src/kleinanzeigen_bot/model/config_model.py

```
@@ -14,6 +14,24 @@
+
+class PriceReductionConfig(ContextualModel):
+    type:Literal["FIXED", "PERCENTAGE"] = Field(
+        description = "type of reduction to apply on each repost: percentage of the previous price or fixed amount"
+    )
+    value:float = Field(
+        gt = 0,
+        description = "magnitude of the reduction; interpreted as percent when type=percentage and as currency units when type=fixed"
+    )
+
+    @model_validator(mode = "after")
+    def _validate_percentage_range(self) -> "PriceReductionConfig":
```

Ruff (0.14.4)

Remove quotes from type annotation

Remove quotes

(UP037)